### ISSUE TITLE
AnLinux.exe version

### ISSUE THREAD

**Issue by Pantyhose-X** [EXALAB/AnLinux-App#344]
I hope there is an AnLinux Win10 version
There is no need to open the [BIOS Intel Virtualization Technology](https://www.sony-asia.com/electronics/support/articles/S500016173). Similar [Bluestacks](https://www.bluestacks.com/) does not require [BIOS Intel Virtualization Technology](https://www.sony-asia.com/electronics/support/articles/S500016173)

**Comment by ShadowEO** [comment]
Ofc you can run any VM you like WITHOUT VT-x, it'll just run slower. Why would you not wish to turn on VT-x?

**Comment by supersonictw** [comment]
AnLinux is based on `proot`, which like UNIX's `chroot`, there is not way to mount Windows 10, because the system kernel is based on NT core, not any Unix-Like core.

https://en.wikipedia.org/wiki/Chroot

**Comment by supersonictw** [comment]
https://github.com/limboemu/limbo

This is the project you hoping to forwarding, the app is an x86 emulator.

**Comment by supersonictw** [comment]
Umm...the issue is relating to your hoping:

https://github.com/EXALAB/AnLinux-App/issues/346

It's possible run Windows ARM/Linux with KVM.

**Comment by Pantyhose-X** [comment]
@supersonictw 
I am talking about compiling an AnLinux.exe version

**Comment by ShadowEO** [comment]
@supersonictw For some reason, he wants a version of this app to run ON Windows 10....

> @supersonictw I am talking about compiling an AnLinux.exe version

You know that the reason AnLinux doesn't need BIOS settings changes is because it's an ARM platform, right? ARM processors that support virtualization, have it enabled, full stop. Once again, what are you looking for that you can't get via Hyper-V, VirtualBox, vmWARE, QEMU, etc. which fulfill the exact same purpose.

If turning on VT-x/VT-d, is such a big deal for you, none of those require it to be on except for Hyper-V and possibly vmWARE. You can run whatever you want, unaccelerated as you please. I don't understand why you'd WANT to run VMs that don't run at full performance but to each their own. o_o.

**Comment by Pantyhose-X** [comment]
> @supersonictw For some reason, he wants a version of this app to run ON Windows 10....
> 
> > @supersonictw I am talking about compiling an AnLinux.exe version
> 
> @Pantyhose-X You know that the reason AnLinux doesn't need BIOS settings changes is because it's an ARM platform, right? ARM processors that support virtualization, have it enabled, full stop. Once again, what are you looking for that you can't get via Hyper-V, VirtualBox, vmWARE, QEMU, etc. which fulfill the exact same purpose.
> 
> If turning on VT-x/VT-d, is such a big deal for you, none of those require it to be on except for Hyper-V and possibly vmWARE. You can run whatever you want, unaccelerated as you please. I don't understand why you'd WANT to run VMs that don't run at full performance but to each their own. o_o.
> 
> What you are asking for in this project, has already been achieved on the platform you're asking about by other projects, which is within their scope. Asking for a Windows version of an Android-based virtualization app is just out of scope for the project, I'd wager.
> 
> TL;DR @Pantyhose-X If you want a linux distribution on your PC in the same vein as this app, turn on Virtualization in the BIOS (suck it up buttercup), and then inside windows, open the run box (WIN+R), type optionalfeatures and hit enter. Turn on Hyper-V, Windows Subsystem for Linux, Windows Hypervisor Platform, etc and hit Ok. Reboot and then install a distribution from the Microsoft Store. Done.

Thanks
